src/components/decimal/definition.js

```diff
@@ -15,6 +15,7 @@ let definition = new Definition('decimal', Decimal, {
 * Entering whole numbers without a decimal point? [Try Number Input](/components/number-input).
  `,
   type: 'form',
+  hiddenProps: ['onBlur', 'onKeyDown'],
```

Comment:
Also fieldHelpInline as its appearing. (although this should probably be removed from the input defintion and added to the checkbox and radio button)